src/openlifu/seg/skinseg.py

```diff
@@ -420,3 +438,45 @@ def spherical_interpolator_from_mesh(
     )
 
     return interpolator
+
+def _spherical_interpolator_from_mesh_embree(surface_mesh : vtk.vtkPolyData) -> Callable:
```

Comment:
this function's implementation doesn't need to be reviewed closely because the unit test `test_spherical_interpolator_from_mesh` covers it well; we now run that test on both `_spherical_interpolator_from_mesh_cell_locator` (the old method) and `_spherical_interpolator_from_mesh_embree` (the new method)